src/seedu/addressbook/Main.java

```diff
@@ -80,6 +80,11 @@ private void runCommandLoopUntilExitCommand() {
         Command command;
         do {
             String userCommandText = ui.getUserCommand();
+            try {
```

Comment:
this may not be the right place to check for the file existence.
the way i foresee it, if you start the program with no file, then the program will throw this exception, which is incorrect use case.

you should be checking for file presence or absence in the `save()` method.